src/app.ts

```diff
@@ -57,7 +57,7 @@ export class DanetApplication {
 		this.controller.abort();
 	}
 
-	listen(port = 3000): Promise<ApplicationListenEvent> {
```

Comment:
Random port as default port is IMO not recommended. It makes it harder to create automated test outside the codebase, for example with postman.